src/components/Synthetics/TradeHistoryRow/helpers.ts

```diff
@@ -0,0 +1,382 @@
+import CustomErrors from "abis/CustomErrors.json";
+import { t } from "@lingui/macro";
+import words from "lodash/words";
+import { StatsTooltipRowProps } from "components/StatsTooltip/StatsTooltipRow";
+import { OrderType, isIncreaseOrderType } from "domain/synthetics/orders";
+import { formatAcceptablePrice } from "domain/synthetics/positions";
+import { convertToUsd } from "domain/synthetics/tokens";
+import { getShouldUseMaxPrice, getTriggerThresholdType } from "domain/synthetics/trade";
+import { PositionTradeAction, TradeAction, TradeActionType } from "domain/synthetics/tradeHistory";
+import { BigNumber, ethers } from "ethers";
+import { PRECISION } from "lib/legacy";
+import { formatDeltaUsd, formatTokenAmount, formatTokenAmountWithUsd, formatUsd } from "lib/numbers";
+import { museNeverExist } from "lib/types";
+import { trimStart } from "lodash";
+
+type FormatPositionMessageChunk = {
+  text: string;
+  textRed?: boolean;
```

Comment:
i'm not sure about textRed naming

1. textRed sounds as a string field
2. is it field to indicate that PositionMessage has some error right? maybe just rename it to isError?